tests/plugins/tabletools/cellproperties.js

```diff
@@ -100,18 +86,7 @@
 		},
 
 		// #11439
-		'test load and update field values (#8)': function() {
-			this.doTest( 'table-9', function( dialog ) {
-				assert.areSame( '', dialog.getValueOf( 'info', 'cellType' ) );
```

Comment:
same here, it might be worth keeping the test for this around as I don't think that was removed

Reply:
Right! I put test back and omitted `bgColor`